packages/netlify-cms-core/package.json

```diff
@@ -62,7 +62,6 @@
     "react-window": "^1.8.5",
     "redux": "^4.0.5",
     "redux-notifications": "^4.0.1",
-    "redux-optimist": "^1.0.0",
```

Comment:
You probably need to run `yarn install` locally and push a new `yarn.lock` file too